MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -71,6 +71,17 @@ open class BaseInstructionsBannerView: UIControl {
         delegate?.didTapInstructionsBanner(self)
     }
     
+    func update(progress: RouteLegProgress, location: CLLocation, secondsRemaining: TimeInterval) {
```

Comment:
`location` and `secondsRemaining` are unused, so  `setRouteProgress(_:)` would suffice as the method name.